packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionId.tsx

```diff
@@ -1,19 +1,16 @@
 import { Row, Text } from '@trezor/components';
 import { spacings } from '@trezor/theme';
 import { Translation } from 'src/components/suite';
-import { CoinmarketTransactionText } from 'src/views/wallet/coinmarket';
 
 interface CoinmarketTransactionIdProps {
     transactionId: string;
 }
 
 export const CoinmarketTransactionId = ({ transactionId }: CoinmarketTransactionIdProps) => (
-    <CoinmarketTransactionText>
+    <Text margin={{ top: spacings.xs }} variant="tertiary" typographyStyle="label">
```

Comment:
You'll have to add as="div" in cases like these, otherwise you'd get error for nesting divs in spans

Reply:
ok